x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -236,3 +236,33 @@ test('Should pluck the categorical style-meta from fieldmeta', async () => {
     ],
   });
 });
+
+test('Should read out categorical type correctly', async () => {
```

Comment:
How about renaming to `isCategorical  should return true when property is categorical`?